frontend/public/components/modals/configure-update-strategy-modal.tsx

```diff
@@ -0,0 +1,170 @@
+import * as _ from 'lodash-es';
+import * as React from 'react';
+import { Tooltip } from '../utils/tooltip';
+
+import { k8sPatch, Patch, DeploymentUpdateStrategy, K8sResourceKind } from '../../module/k8s';
+import { DeploymentModel } from '../../models';
+import { createModalLauncher, ModalTitle, ModalBody, ModalSubmitFooter } from '../factory/modal';
+import { pluralize, withHandlePromise } from '../utils';
+import { RadioInput } from '../radio';
+
+const getNumberOrPercent = (value) => {
+  if (typeof value === 'undefined') {
+    return null;
+  }
+  if (typeof value === 'string' && value.indexOf('%') > -1) {
+    return value;
+  }
+
+  return _.toInteger(value);
+};
+
+export const ConfigureUpdateStrategy: React.FC<ConfigureUpdateStrategyProps> = (props) => {
+  return <React.Fragment>
+    <div className="co-m-form-row">
+      <p>
+        How should the pods be replaced when a new revision is created?
+      </p>
+    </div>
+
+    <div className="row co-m-form-row">
+      <div className="col-sm-12">
+        <RadioInput
+          onChange={e => props.onChangeStrategyType(e.target.value)}
+          value="RollingUpdate"
+          checked={props.strategyType === 'RollingUpdate'}
+          title="RollingUpdate"
+          subTitle="(default)"
+          autoFocus={props.strategyType === 'RollingUpdate'}>
+          <div className="co-m-radio-desc">
+            <p className="text-muted">
+              Execute a smooth roll out of the new revision, based on the settings below
+            </p>
+
+            <div className="row co-m-form-row">
+              <div className="col-sm-3">
+                <label htmlFor="input-max-unavailable" className="control-label">
+                  Max Unavailable
+                </label>
+              </div>
+              <div className="co-m-form-col col-sm-9">
+                <div className="form-inline">
+                  <div className="input-group">
+                    <input disabled={props.strategyType !== 'RollingUpdate'}
+                      placeholder="25%" size={5} type="text" className="form-control"
```

Comment:
Not sure if you're done updating, but there are still instances of `form-control` that need to be switched over to `pf-c-form-control`.